exercises/dnd-character/example.exs

```diff
@@ -0,0 +1,29 @@
+defmodule DndCharacter do
+  defstruct [
+    :strength,
+    :dexterity,
+    :constitution,
+    :intelligence,
+    :wisdom,
+    :charisma,
+    :hitpoints
+  ]
+
+  def modifier(score), do: floor((score - 10) / 2)
```

Comment:
needs to be `:math.floor`

Reply:
Not sure on this. I intended to use [`Kernel.floor/1`](https://hexdocs.pm/elixir/Kernel.html#floor/1), and that should be available.

Reply:
Kernel.floor is only available in >= 1.8

Reply:
Okay. Well, I'll go figure out a 1.6.6 solution.